client/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.core.validators import MinLengthValidator
+
+
+class User(models.Model):
+    user_name = models.CharField(max_length=32, validators=[MinLengthValidator(6)], primary_key=True, null=False)
+    first_name = models.CharField(max_length=32, validators=[MinLengthValidator(1)])
+    last_name = models.CharField(max_length=32, validators=[MinLengthValidator(1)])
+    password = models.CharField(max_length=16, validators=[MinLengthValidator(6)])
+    user_type = models.CharField(max_length=16, validators=[MinLengthValidator(6)],
+                                 choices=[('Client', 'Client'), ('Supplier', 'Supplier')])
```

Comment:
ive added choices field to but did it like this:
`choices=[('client'),('supplier')]`
i think it will be better to have one option for each, that way filtering the field will have less options -> less bugs.

Reply:
> ive added choices field to but did it like this: `choices=[('client'),('supplier')]` i think it will be better to have one option for each, that way filtering the field will have less options -> less bugs.

I wrote it like that because this is the right syntax. you can try it and see how it looks in your admin. 
it should stay how I wrote it.